addons/sky_3d/src/Sky3D.gd

```diff
@@ -151,9 +152,17 @@ func set_update_interval(value:float) -> void:
 	if tod:
 		tod.update_interval = value
 
+func set_manual_time_management(value:bool) -> void:
+	manual_time_management = value
+	if tod:
+		tod._manual_time_management = value
+
+func manual_time_update() -> void:
+	if tod:
+		tod.manual_time_update()
 
 func _on_timeofday_updated(time: float) -> void:
-	if tod and Engine.is_editor_hint():
+	if tod and (Engine.is_editor_hint() or manual_time_management):
 		minutes_per_day = tod.total_cycle_in_minutes
 		current_time = tod.total_hours
 		update_interval = tod.update_interval
```

Comment:
Revert. Here the editor hint is appropriate as its only function is to update value from TOD.